yt/frontends/ramses/data_structures.py

```diff
@@ -795,6 +795,12 @@ class RAMSESDataset(Dataset):
     _field_info_class = RAMSESFieldInfo
     gamma = 1.4  # This will get replaced on hydro_fn open
 
+    # RAMSES-specific parameters
+    force_cosmological: Optional[bool]
+    _force_max_level: tuple[int, str]
+    _bbox: Optional[list[list[float]]]
+    self_shielding: Optional[bool]
```

Comment:
Setting this to None will fix the failing `load_archive` test 
```suggestion
    self_shielding: Optional[bool] = None
```